polling/src/main/java/io/atleon/polling/AloPollingReceiver.java

```diff
@@ -0,0 +1,64 @@
+package io.atleon.polling;
+
+import io.atleon.core.Alo;
+import io.atleon.core.AloFactory;
+import io.atleon.core.AloFlux;
+import io.atleon.core.OrderManagingAcknowledgementOperator;
+import io.atleon.polling.reactive.Pollable;
+import io.atleon.polling.reactive.PollerOptions;
+import io.atleon.polling.reactive.PollingReceiver;
+import io.atleon.polling.reactive.PolledRecord;
+import io.atleon.polling.reactive.PollingSourceConfig;
+import io.atleon.polling.reactive.ReceiverRecord;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Sinks;
+import reactor.core.scheduler.Schedulers;
+
+public class AloPollingReceiver<P, O> {
+
+    private final Pollable<P, O> pollable;
+    private final PollingSourceConfig config;
+
+    private AloPollingReceiver(final Pollable<P, O> pollable,
+                               final PollingSourceConfig config) {
+        this.pollable = pollable;
+        this.config = config;
+    }
+
+    public static <P, O> AloPollingReceiver<P, O> from(final Pollable<P, O> pollable,
+                                                       final PollingSourceConfig config) {
+        return new AloPollingReceiver<>(pollable, config);
+    }
+
+    public AloFlux<P> receiveRecords() {
+        return Flux.defer(() -> receiveAloRecords()
+                        .map(PolledRecord::getPayload))
+                .as(AloFlux::wrap);
```

Comment:
Can be simplified to 
```java
return Flux.defer(this::receiveAloRecords)
    .map(PolledRecord::getPayload)
```

After writing that, I wonder if this method should rather be `receivePayloads()` 🤔